skate-plugin/src/main/resources/messages/skateBundle.properties

```diff
@@ -0,0 +1,3 @@
+skate.configuration.title=Skate Settings
+skate.configuration.baselineFile.title=Baseline file:
```

Comment:
Let's fix these names, as they're not baseline files

Reply:
Here is the commit where I changed the name: 22729e86ce009f741e96202899dd81dff1f077e6